spec/features/unlink_court_applications_spec.rb

```diff
@@ -36,6 +36,26 @@
       expect(page).to have_content "Enter the MAAT ID"
     end
 
+    scenario 'I send correct params to CDA' do
+      unlink_stub = stub_request(:patch, /.*court_application_laa_references.*/).with(
+        body: {
+          laa_reference: {
+            subject_id: "6c3eded6-a6d6-4156-940d-e3b5f02deb96",
+            user_name: "kova-a81",
+            unlink_reason_code: 4,
+            unlink_other_reason_text: nil,
+            maat_reference: "1234568"
+          }
+        }.to_json
```

Comment:
this test is already using VCR,
my concern is that it'd be even more difficult to maintain if we start mixing VCR and stub_request 🤔